mingw-w64-fortran-stdlib/PKGBUILD

```
@@ -0,0 +1,42 @@
+# Maintainer: ZUO Zhihua <zuo.zhihua@qq.com>
+
+_realname=fortran-stdlib
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.2.1
+pkgrel=1
+pkgdesc="Fortran Standard Library"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64')
+url="https://github.com/fortran-lang/stdlib"
+license=('MIT')
+provides=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
+         "${MINGW_PACKAGE_PREFIX}-libwinpthread"
+         "${MINGW_PACKAGE_PREFIX}-gcc-libgfortran")
```

```suggestion
         $([[ ${MINGW_PACKAGE_PREFIX} == *-clang-* ]] || echo "${MINGW_PACKAGE_PREFIX}-gcc-libgfortran"))
```
Actually, clang64 and clangarm64 have flang compiler.

---

Yes, the clang64 environment has a flang compiler, but it's not perfect yet, only supports the Fortran 1990 standard, and fortran-stdlib has a minimum Fortran standard of 2008, so flang doesn't compile fortran-sdtlib yet.

Finally, thanks for reviewing.